deploy/sql/anonymise-sql-data/pupil-anon.sql

```diff
@@ -1,8 +0,0 @@
-UPDATE mtc_admin.pupil SET 
-    foreName=CAST(NEWID() AS VARCHAR(255)),
```

Comment:
gut feel is we should save these anonymisation scripts as they may come in useful in the future.  maybe add under a misc section in the db app?